backend/_handler.js

```diff
@@ -4,106 +4,110 @@ import * as db from '@reshuffle/db';
   } catch (e) {
-      res.sendStatus(500);
-      console.error(e);
+    res.sendStatus(500);
+    console.error(e);
   }
 });
 
 //GET /collection/:id - gets just one entry. This is the proper way to get a single ID and not using a filter command.
 app.get('/api/pets/:id', express.json(), async (req, res) => {
   try {
-      const id = req.params.id;
-      var key = `pets/${id}`;
-      const result = await db.get(key);
-      
-      res.setHeader('Content-Type', 'application/json');
-      res.end(JSON.stringify(result));
+    const id = req.params.id;
+    var key = `pets/${id}`;
+    const result = await db.get(key);
+
+    res.setHeader('Content-Type', 'application/json');
+    res.end(JSON.stringify(result));
   } catch (e) {
-      res.sendStatus(500);
-      console.error(e);
+    res.sendStatus(500);
+    console.error(e);
   }
 });
 
 //PUT /collection/:id - updates an entry. Should return the updated entry.
 app.put('/api/pets/:id', express.json(), async (req, res) => {
   try {
-      var value = req.body;
-      var id = req.params.id;
-      value.id = id;
-      var oldEntry = await db.get(`pets/${id}`);
-      if(oldEntry){
-        var response =  await db.update(`pets/${id}`, (prev_value) => { return value; });
-        res.setHeader('Content-Type', 'application/json');
-        res.end(JSON.stringify(oldEntry));
-      }else{
-        res.sendStatus(204);
-      }
-      
+    var value = req.body;
+    var id = req.params.id;
+    value.id = id;
+    var oldEntry = await db.get(`pets/${id}`);
+    if (oldEntry) {
+      var response = await db.update(`pets/${id}`, (prev_value) => {
+        return value;
+      });
+      res.setHeader('Content-Type', 'application/json');
+      res.end(JSON.stringify(oldEntry));
+    } else {
+      res.sendStatus(204);
+    }
   } catch (e) {
-      res.sendStatus(500);
-      console.error(e);
+    res.sendStatus(500);
+    console.error(e);
   }
-}); 
+});
 
 //DELETE /collection/:id - I’ll let you figure that one out on your own :) should return a 204 no content if successful.
 app.delete('/api/pets/:id', express.json(), async (req, res) => {
   try {
-      const id = req.params.id;
-      if (!id) {
-        res.sendStatus(400);
-    }else{
-        const result = await db.remove(`pets/${id}`);
-        res.sendStatus(204);
-    }  
+    const id = req.params.id;
+    if (!id) {
+      res.sendStatus(400);
+    } else {
+      const result = await db.remove(`pets/${id}`);
+      res.sendStatus(204);
+    }
   } catch (e) {
-      res.sendStatus(500);
-      console.error(e);
+    res.sendStatus(500);
+    console.error(e);
   }
 });
 
-// other endpoints and funtions 
-
-function uuidv4() {
+/**
+ * Generate unique IDentifier
+ */
+function generateUid() {
   return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function(c) {
-    var r = Math.random() * 16 | 0, v = c == 'x' ? r : (r & 0x3 | 0x8);
+    var r = (Math.random() * 16) | 0,
```

Comment:
Why not `import uuidv4 from 'uuid/v4';`?

Reply:
+1 for using proper uuid library

also, the code uses `var` in many places. pick one style and stick to it (i suggest `const` and `let`)